packages/react-component-library/src/helpers.ts

```diff
@@ -0,0 +1,9 @@
+function warnIfOverwriting<P>(props: P, propertyName: string, componentName: string) {
```

Comment:
Not for now - but maybe as the library evolves we could wrap console.log so we can enable / disable logging based on message types / change verbosity. Something to think about in the future - might be completely overkill.